src/sessions/video-session-handler.ts

```diff
@@ -255,6 +287,11 @@ export class VideoSessionHandler extends BaseSessionHandler {
       return;
     }
 
+    // auto answer agent sessions
+    if (isAgentVideoJid(pendingSession.fromJid)) {
+      await super.handlePropose(pendingSession);
```

Comment:
Would it be worth adding a log for this situation?

Reply:
I agree with Jon and think we should add a log here.